### ISSUE TITLE
Is There Any Benchmark for the Consumer?

### ISSUE THREAD

**Issue by sparkthu** [wvanbergen/kafka#108]
I run the example, and find it very slow. I'm not sure where the bottleneck is.

**Comment by wvanbergen** [comment]
No. It has always been fast enough for my purposes, so I never bothered benchmarking or profiling. I'd happy if someone would provide some benchmarking or profiling numbers.

I do know that the re-balancing consumers is unnecessarily slow. I attempted to address it in #72 but I never followed through on finishing it.
